### PR TITLE
react1-week3

### PR DESCRIPTION
**Pull request by Hadis-jamali**

### CODE REVIEW

File: frontend/src/components/MealsList.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+function MealsList() {
+  const [meals, setMeals] = useState();
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get("http://127.0.0.1:5000/api/meals");
+        if (response.data !== null) {
+          console.log(response.data);
```
Comment (BrantalikP) [inline frontend/src/components/MealsList.jsx:14]: as this is an assignment its fine, but keep in mind it's not a good practice to leaving unnecessary console logs in production code :)

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import MealsList from './components/MealsList'
+function App() {
+  return (
+    <div>
```
Comment (BrantalikP) [inline frontend/src/App.jsx:4]: unnecessary div, you can delete it :)

File: frontend/src/components/MealItem.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "../index.css";
+function MealItem({ meal }) {
+  return (
+    <div >
+      <img />
```
Comment (BrantalikP) [inline frontend/src/components/MealItem.jsx:6]: what is the purpose of having an empty image tag?

**Review by BrantalikP — CHANGES_REQUESTED** [review]
Except few small things, its good, well done @Hadis-jamali 👏

File: frontend/src/components/MealItem.jsx (hunk 3 above)
Comment (Hadis-jamali) [reply frontend/src/components/MealItem.jsx:6]: Hi petr , I want to add image for header and i didn't have time!😃 I added header for this week homework.

Comment (BrantalikP) [reply frontend/src/components/MealItem.jsx:6]: ok I see, so if its not a part of the next one, then please remove it. Thank you! :)

**Pull request merged**